int16 bb_number
scaled_yolov4/BB[] bb_array